synchronization/src/cyface/java/de/cyface/synchronization/CyfaceAuthenticator.java

```diff
@@ -125,7 +134,7 @@ public Bundle getAuthToken(final @Nullable AccountAuthenticatorResponse response
 
                 // Get Auth token
                 try {
-                    authToken = initSync(account.name, password, sslContext);
+                    authToken = initSync(account.name, password, sslContext); // HOW TO GET THE AUTHORITY HERE ?
```

Comment:
How was it done previously?

Reply:
deprecated comment of mine, ignore it (removed it)